README.rst

```diff
@@ -27,7 +27,7 @@ The recommended way to install Silex is through `Composer`_:
 
 .. code-block:: bash
 
-    php composer.phar require silex/silex "~1.2"
+    php composer.phar require silex/silex "~1.3"
```

Comment:
``composer require`` to be consistent with the Symfony documentation which always uses the global setup